bitnami/harbor/templates/_helpers.tpl

```diff
@@ -101,6 +101,42 @@ We truncate at 63 chars because some Kubernetes name fields are limited to this
   {{- end -}}
 {{- end -}}
 
+{{- define "harbor.database.clairUsername" -}}
+  {{- if eq .Values.postgresql.enabled true -}}
+    {{- printf "%s" "postgres" -}}
```

Comment:
Shouldn't this be: 
```suggestion
    {{- .Values.postgresql.postgresqlUsername -}}
```

The same for the rest if you think this change is appropriate.

Reply:
You're right. I missed that one, when porting this PR from the official helm chart.
The same applies to `notaryServerUsername` and `notarySignerUsername`.